juspay/hyperswitch
# refactor(events): Adding changes to type of API events to Kafka

## Type of Change



## Description

Making changes in the type of few keys for the API events which are being pushed to Kafka
- Update the AuthenticationType to be Snake case
- stringify the req/res parameters before logging

### Additional Changes





## Motivation and Context



## How did you test it?

- create an api request (e.g merchantaccountcreate)
- Check the loki logs for the api events
- the below authentication type would be in proper snake case

e.g `merchant_jwt` instead of `merchant_j_w_t`

<img width="1545" alt="Screenshot 2023-11-02 at 2 20 14 PM" src="https://github.com/juspay/hyperswitch/assets/21202349/2638c3ab-0aff-4799-bdb1-ad22e8985202">


## Checklist
